datatypes/src/raster/empty_grid.rs

```diff
@@ -16,19 +16,19 @@ use snafu::ensure;
 
 #[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct NoDataGrid<D, T> {
+pub struct EmptyGrid<D, T> {
     pub shape: D,
     pub no_data_value: T,
 }
 
-pub type NoDataGrid1D<T> = NoDataGrid<GridShape1D, T>;
-pub type NoDataGrid2D<T> = NoDataGrid<GridShape2D, T>;
-pub type NoDataGrid3D<T> = NoDataGrid<GridShape3D, T>;
+pub type NoDataGrid1D<T> = EmptyGrid<GridShape1D, T>;
+pub type NoDataGrid2D<T> = EmptyGrid<GridShape2D, T>;
+pub type NoDataGrid3D<T> = EmptyGrid<GridShape3D, T>;
```

Comment:
either `EmptyGrid` and `EmptyGrid1D' (etc.) or `NoDataGrid1D` and `NoDataGrid`

Reply:
03e07cc